camel/messages/base.py

```diff
@@ -21,7 +25,9 @@
     OpenAIUserMessage,
 )
 from camel.prompts import CodePrompt, TextPrompt
-from camel.types import OpenAIBackendRole, RoleType
+from camel.types import OpenAIBackendRole, OpenAIImageType, RoleType
+
+IMAGE_DETAIL = Literal["auto", "low", "high"]
 
```

Comment:
Coule you please add one code comment to explain the meanings for every value very quickly to the users?

Reply:
Additionally, to ensure that the value of IMAGE_DETAIL is one of ["auto", "low", "high"], it's recommended to use an Enum:
```
from enum import Enum

class ImageDetail(Enum):
    AUTO = "auto"
    LOW = "low"
    HIGH = "high"
```

Reply:
During the subsequent code review, I noticed that there was extensive use of terms like "auto," "low," etc. Therefore, for the sake of easier modifications in the future, please use enum types.